website/docs/commands/etag-commands.md

```diff
@@ -0,0 +1,109 @@
+---
```

Comment:
We have a section for Garnet specific commands here, perhaps move this documentation there? Or link to it? https://microsoft.github.io/garnet/docs/commands/garnet-specific-commands

Reply:
got it